layer/trace_layer.cpp

```diff
@@ -53,6 +53,58 @@ const VkLayerProperties kLayerProps = {
 
 const std::vector<VkExtensionProperties> kDeviceExtensionProps = { VkExtensionProperties{ "VK_EXT_tooling_info", 1 } };
 
+/// An alphabetical list of device extensions which we do not report upstream if
+/// other layers or ICDs expose them to us.
+const char* const kDeviceExtensionBlacklist[] = {
+    // Look problematic:
```

Comment:
I added a few more extensions just to exercise the code that runs through this list but these ones I chose do genuinely seem a problem for trace/replay to me.
@bradgrantham-lunarg Should I cull them for now and maybe make a separate issue for expanded the list back out with all the extensions we don't support the tracing of?

Reply:
Yes, I think make a single separate "task" issue listing these and we can then go address each of these in turn if they become a priority.  It looks like most of these are very special-case and we probably will see them go by as a capture issue if they become more important.

Reply:
I'll do that for now. It's my assertion that we should block any extension we can see won't capture right rather than letting users stumble over them, report issues that may not reference the extension in question, and then have to back-track to the root cause. Better for users to see some "required extension missing message" up-front during capture.